src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -545,19 +546,60 @@ private async Task<DurableConfig> ReadDurableTaskConfig()
         // This is a stopgap approach to get the Durable extension version. It duplicates some logic in ExtensionManager.cs.
         private async Task<string> GetDurableMajorVersionAsync(JObject hostJson, ScriptJobHostOptions hostOptions)
```

Comment:
What is the plan for addressing https://github.com/Azure/azure-functions-host/issues/5390? We really need to get this logic out of the host. Durable is the only extension that has custom logic on this path.